.github/workflows/wheel_linux_aarch64.yml

```diff
@@ -32,6 +32,8 @@ jobs:
         exec_model: ${{ fromJson(needs.set_wheel_build_matrix.outputs.exec_model) }}
         kokkos_version: ${{ fromJson(needs.set_wheel_build_matrix.outputs.kokkos_version) }}
         container_img: ["quay.io/pypa/manylinux2014_aarch64"]
+        exclude:
+          - exec_model: OPENMP
```

Comment:
The `exec_model` is still `fromJson(needs.set_wheel_build_matrix.outputs.exec_model)`. Since we are just building `lightning_qubit` on Linux ARM for now, I'm skipping the OPENMP backend and just building with SERIAL as before.